test/test_aggregate_graphs.py

```diff
@@ -122,7 +122,7 @@ def setUp(self):
             graph.parse(StringIO(n3Str), format="n3")
 
         self.graph4 = Graph(memStore, RDFS)
-        self.graph4.parse(RDFS.uri)
+        self.graph4.parse(RDFS)
```

Comment:
Won't this work fine as it was? And does it make sense to change it? Maybe can add both with and without .uri